.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - '6'
-  - '4'
-  - '0.12'
-  - '0.10'
+  - 4
+  - 6
+  - 7
```

Comment:
YAML interprets them as number, but versions are not number in this sense. They should be quoted for consistency. It's only a quirk that they work now.